Source/WebKit/WebProcess/WebPage/Cocoa/WebCookieJarCocoa.mm

```diff
@@ -97,7 +97,8 @@
     if (!cookie || ![[cookie name] length] || [cookie isHTTPOnly])
         return;
 
-    [ensurePartitionedCookieStorage() _setCookies:@[cookie.get()] forURL:cookieURL.createNSURL().get() mainDocumentURL:firstPartyURL.createNSURL().get() policyProperties:policyProperties(sameSiteInfo, cookieURL).get()];
+    RetainPtr partitionedCookieStorage = ensurePartitionedCookieStorage();
```

Comment:
This seems needed because we can't inline `ensurePartitionedCookieStorage()`, but we could `const` `m_partitionedStorageForDOMCookies` for clarity.